#! /bin/sh /usr/share/dpatch/dpatch-run
## fpconst.dpatch by Stefano Zacchiroli <zack@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: import fpconst, which is essential for SOAPpy

@DPATCH@
diff -urNad python-soappy-0.12.0~rc1~/SOAPpy/__init__.py python-soappy-0.12.0~rc1/SOAPpy/__init__.py
--- python-soappy-0.12.0~rc1~/SOAPpy/__init__.py	2005-02-09 16:21:04.000000000 +0100
+++ python-soappy-0.12.0~rc1/SOAPpy/__init__.py	2007-03-24 01:50:37.000000000 +0100
@@ -5,6 +5,7 @@
 from Client      import *
 from Config      import *
 from Errors      import *
+import fpconst
 from NS          import *
 from Parser      import *
 from SOAPBuilder import *
